terms/core.md

```diff
@@ -1,54 +1,61 @@
-**Introduction**
+# Responsible Disclosure Policy
```

Comment:
I'd probably change this to `Core Terms` if we want to add the overall header here

Reply:
Yep, that makes sense.  I'll get that udpated